python/ray/_private/ray_constants.py

```diff
@@ -453,6 +453,15 @@ def env_set_by_user(key):
 RAY_INTERNAL_NAMESPACE_PREFIX = "_ray_internal_"
 RAY_INTERNAL_DASHBOARD_NAMESPACE = f"{RAY_INTERNAL_NAMESPACE_PREFIX}dashboard"
 
```

Comment:
> It bears all environment variables from the JobSupervisor

Actually, should entrypoint process inherit environment variables from the JobSupervisor in the first place?

Reply:
no idea. what's your opinion?

Reply:
We obviously want to bear image-wide envs, but we don't want "suspicious extra envs added by Ray", so we can just do what we do in this PR.